### PR TITLE
Mobile header popups fixed

### PR DESCRIPTION
**Pull request by Agbeleshe**
I created a CSS class to send overlapping component to the back of the modal ("send-to-back") as seen below. 
also made the background image to be at the back of the entire page by creating a CSS component, as also seen below ("send-to-ultimate-back") .

### CODE REVIEW

File: frontend/src/components/mobDropdownMenu/mobDropdownMenu.css [hunk 1]
```diff
@@ -19,12 +19,22 @@
...
+  .send-to-back {
+    z-index: -5;
+    position: relative;
+  }
+
+  .send-to-ultimate-back {
+    z-index: -10;
+    position: relative;
+  }
+
```
Comment (whateverfw) [inline frontend/src/components/mobDropdownMenu/mobDropdownMenu.css:37]: this solution requires to assign those classes on each new page and to multipli elements, which is not ideal
Please create a solution that does not involve changes in other pages/component apart from dropdown one.

Solution should be applied to both (2) dropdowns

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/components/mobDropdownMenu/mobDropdownMenu.css (hunk 1 above)
Comment (Agbeleshe) [reply frontend/src/components/mobDropdownMenu/mobDropdownMenu.css:37]: Alright, i'm on it.

**Comment by whateverfw** [comment]
@Agbeleshe any updates?

**Comment by djeck1432** [comment]
@Agbeleshe any updates? you have 6 hours to finish task otherwise you will get partially reward

**Comment by djeck1432** [comment]
@Agbeleshe Any updates?

**Comment by Agbeleshe** [comment]
Yes, Sorry for the delay will soon make a PR

**Comment by djeck1432** [comment]
@Agbeleshe okay, please don't hesitate, your task is very  important we need to close it today

**Comment by djeck1432** [comment]
@Agbeleshe unfortunately your task will be in upcoming release, so please, finish your task today

**Comment by djeck1432** [comment]
@Agbeleshe any updates?

**Comment by djeck1432** [comment]
sorry, texted you in DM in telegram and pinged you here, any updates, we should unassigned you @Agbeleshe

**Comment by djeck1432** [comment]
sorry, texted you in DM in telegram and pinged you here, any updates, we should unassigned you @Agbeleshe

**Pull request closed without merging**